packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -514,6 +514,42 @@ describe('Test starting different modes', () => {
 		expect(themeName.text).toContain('Twenty Twenty-Three');
 	});
 
+	/**
+	 * Test that startWPNow compresses the text files.
+	 */
+	test('startWPNow compresses the text files', async () => {
+		const projectPath = path.join(tmpExampleDirectory, 'theme-with-assets');
+		const config = await getWpNowConfig({ path: projectPath });
+		const { options } = await startWPNow(config);
```

Comment:
After debugging the tests, I found that error.
`startWPNow` doesn't start the express server, it only prepares the `php-wasm` instance.
We should call to `startServer` where your middleware was implemented.

```suggestion
		const { options } = await startServer(config);
```

Reply:
@sejas maybe using a clearer distinction between the two functions would reduce the future confusion? WPNow sounds like it is the server after all.